nvflare/app_opt/confidential_computing/cc_manager.py

```diff
@@ -11,158 +11,240 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import threading
+import time
+from typing import Dict, List
 
 from nvflare.apis.event_type import EventType
 from nvflare.apis.fl_component import FLComponent
-from nvflare.apis.fl_constant import AdminCommandNames, FLContextKey
+from nvflare.apis.fl_constant import FLContextKey, RunProcessKey
 from nvflare.apis.fl_context import FLContext
 from nvflare.apis.fl_exception import UnsafeComponentError
-
-from .cc_helper import CCHelper
+from nvflare.app_opt.confidential_computing.cc_authorizer import CCAuthorizer, CCTokenGenerateError, CCTokenVerifyError
+from nvflare.fuel.hci.conn import Connection
+from nvflare.private.fed.server.training_cmds import TrainingCommandModule
 
 PEER_CTX_CC_TOKEN = "_peer_ctx_cc_token"
 CC_TOKEN = "_cc_token"
+CC_ISSUER = "_cc_issuer"
+CC_NAMESPACE = "_cc_namespace"
 CC_INFO = "_cc_info"
 CC_TOKEN_VALIDATED = "_cc_token_validated"
+CC_VERIFY_ERROR = "_cc_verify_error."
+
+CC_ISSUER_ID = "issuer_id"
+TOKEN_GENERATION_TIME = "token_generation_time"
+TOKEN_EXPIRATION = "token_expiration"
+
+SHUTDOWN_SYSTEM = 1
+SHUTDOWN_JOB = 2
+
+CC_VERIFICATION_FAILED = "not meeting CC requirements"
 
 
 class CCManager(FLComponent):
-    def __init__(self, verifiers: list):
+    def __init__(
+        self,
+        cc_issuers_conf: [Dict[str, str]],
+        cc_verifier_ids: [str],
+        verify_frequency=600,
+        critical_level=SHUTDOWN_JOB,
```

Comment:
I thought about this more descriptive way of config initially. The only thing I didn't take it is because the "action_for_failed_verification" name does not sound so well. I couldn't come up with a better meaningful name.

But I am open with this.